controllers/userQuiz.js

```diff
@@ -57,4 +57,20 @@ const addUserQuiz = async (user, quiz, startTime, endTime) => {
     return await getUserQuiz(userQuiz.id)
 }
 
-export { addUserQuiz, getUserQuiz, getUserQuizzes, getAllUserQuizzes }
+const setUserQuizScore = async (userQuizID, score) => {
+    let obj = await getUserQuiz(userQuizID)
```

Comment:
name it properly thx

Reply:
Sorry, name the function again or obj?

Reply:
I think he meant the variable name 'obj'